src/Service.Tests/SqlTests/RestApiTests/Patch/PatchApiTestBase.cs

```diff
@@ -589,6 +589,52 @@ public virtual async Task PatchOneWithNonNullableFieldAsNull()
             );
         }
 
+        /// <summary>
+        /// Verifies that we throw an exception when an extraneous field provided in the request body for a PATCH operation
+        /// does not map to a valid backing column in the table.
+        /// </summary>
+        [TestMethod]
+        public async Task PatchOneTestWithExtraneousFieldsInRequestBody()
+        {
+            // Non-existing field 'hazards' included in the request body for a table with non-autogenerated PK.
```

Comment:
can you note the significance of testing against entity with non-autogenerated versus autogenerated PK?

Reply:
No difference actually, yes. Can keep one test only.